nixos/modules/services/web-apps/rimgo.nix

```diff
@@ -0,0 +1,86 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
```

Comment:
```suggestion
let
```

Reply:
should i replace the `with lib;` with an `inherit (lib) ...` to make the output more predictable?

Reply:
Yes.